src/components/accordion/default/index.njk

```diff
@@ -0,0 +1,26 @@
+---
+title: Accordion
+layout: layout-example.njk
+---
+
+{% from "accordion/macro.njk" import govukAccordion %}
+
+{{ govukAccordion({
+  id: "accordion-default",
+  items: [
+    {
+      label: "Section A",
+      content: {
+        html: "<ul class='govuk-list govuk-list--bullet'><li>Example item 1</li>
```

Comment:
Do we have a more real world example for these code examples?

The content is a bit placeholder.